### ISSUE TITLE
The workflow generators should write a veto definer file for reproducatibility

### ISSUE THREAD

**Issue by duncan-brown** [ligo-cbc/pycbc#1481]
If the veto definer file contains a segment type with a zero end time, then it is possible that two runs of the workflow will not produce the same results if someone has changed the segments in the database.

For reproducibility, the workflow generator should write out a veto definer file with start and end times populated based on the segment summary table in the database *at the time the workflow is built* to allow reproducibility.

**Comment by ahnitz** [comment]
@duncan-brown This seems like a good idea. Is it worth actually having a check, like Ian has done for a lot of other segment issues, that sees if the veto has been defined for the entire period of the analysis?

**Comment by tdent** [comment]
Is this still an issue?

**Comment by lauranuttall** [comment]
I don't think this is an issue anymore. There are ways to track which veto-definer is used.

**Comment by tdent** [comment]
@lauranuttall @duncan-brown If you don't think it's an issue, please close.
